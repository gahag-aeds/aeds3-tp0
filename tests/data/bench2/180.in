180 90
17 77 19 71 66 17 88 59 71 47 52 49 42 39 24 10 66 25 56 67 67 12 22 57 71 86 52 93 94 11 64 11 40 35 83 6 52 71 65 23 70 69 24 12 9 1 75 75 26 31 43 45 95 17 2 19 3 54 12 98 17 28 9 9 63 44 15 67 67 33 90 37 54 66 2 63 67 77 91 93 60 86 90 8 3 92 27 58 98 39 56 67 67 18 76 82 14 44 49 82 29 91 71 83 57 73 47 25 2 90 70 63 76 61 71 79 5 50 37 56 89 46 23 8 16 52 42 30 48 91 12 77 34 84 12 91 9 59 68 12 49 39 27 77 52 50 56 57 0 46 13 41 92 89 1 60 41 43 90 41 86 55 18 20 39 30 63 0 42 32 12 43 23 39 21 75 41 29 84 41 
Sub 28 87
Sum 51 110
Add 115 174
Sum 72 131
Sub 72 131
Sum 96 155
Sub 72 131
Min 73 132
Add 119 178
Min 43 102
Sub 94 153
Sum 117 176
Sub 58 117
Min 64 123
Add 57 116
Sum 9 68
Sub 46 105
Min 1 60
Add 49 108
Sum 48 107
Add 93 152
Max 62 121
Sub 64 123
Max 68 127
Add 93 152
Min 37 96
Add 63 122
Sum 27 86
Sub 46 105
Min 19 78
Sub 28 87
Max 74 133
Add 69 128
Max 23 82
Add 25 84
Max 20 79
Sub 110 169
Min 97 156
Sub 84 143
Min 109 168
Add 11 70
Min 49 108
Add 105 164
Max 68 127
Sub 104 163
Min 34 93
Sub 68 127
Sum 30 89
Sub 34 93
Max 116 175
Add 69 128
Min 118 177
Add 49 108
Sum 12 71
Add 57 116
Max 14 73
Sub 48 107
Max 119 178
Sub 40 99
Max 86 145
Add 9 68
Max 59 118
Add 31 90
Sum 78 137
Add 81 140
Max 47 106
Add 89 148
Sum 63 122
Add 23 82
Sum 45 104
Add 51 110
Min 25 84
Add 85 144
Sum 27 86
Sub 84 143
Sum 69 128
Sub 52 111
Min 31 90
Sub 90 149
Max 77 136
Add 27 86
Max 38 97
Sub 66 125
Sum 75 134
Sub 42 101
Max 2 61
Sub 88 147
Sum 81 140
Sub 112 171
Max 119 178
