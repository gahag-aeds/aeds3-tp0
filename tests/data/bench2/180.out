3027
3260
3004
-1
-1
2662
-2
2555
-1
2955
96
95
-2
2721
-2
95
95
96
2
1
-2
95
-2
2647
92
1
2593
98
92
94
97
3201
98
3312
2958
0
2864
3343
-1
97
100
3231
99
3039
91
